.gitignore

```diff
@@ -37,4 +37,6 @@ database.trace.db
 # Ignore Mac OS X
 **/.DS_Store
 .idea/
+.vscode/*
+!.vscode/launch.json
```

Comment:
I put these lines in the .gitignore to ignore some Java format configuration files.